python/src/aac/__init__.py

```diff
@@ -12,7 +12,7 @@
 import logging
 import os
 
-__version__ = "0.2.7"
```

Comment:
We only updated a dev and testing dependency, it shouldn't impact users. I wouldn't even consider it a breaking change unless I'm overlooking something.

Reply:
Ah shoot. I tend to consider dependency changes breaking because someone could be using things in a way we're not testing. That said, given that these are dev and test dependencies I'm okay bumping the minor version or not considering it a meaning change. Honestly, I didn't pay much attention when I decided that... 😳